staging/src/k8s.io/kube-aggregator/pkg/apis/apiregistration/types.go

```diff
@@ -83,6 +83,32 @@ type APIServiceSpec struct {
 	// version, then minor version. An example sorted list of versions:
 	// v10, v2, v1, v11beta2, v10beta3, v3beta1, v12alpha1, v11alpha2, foo1, foo10.
 	VersionPriority int32
+
+	// ScopedResources declares a range of scoped resources to tell what resources to provide by the APIService. there can be more resources provided
+	// by the servicebackend. k8s aggregator focuses on the resources declared by ScopedResources, if it is defined.
+	ScopedResources []ScopedResource `json:"scopedResources,omitempty" protobuf:"bytes,7,opt,name=scopedResources"`
```

Comment:
the internal version doesn't have the field tags. the internal version is the intermediate to do conversions across external versions. please remove them. the same to other structs/fields.